examples/example-gateway/build.yaml

```diff
@@ -4,6 +4,9 @@ copyrightHeader: ./copyright_header.txt
 endpointConfig: ./endpoints
 genCodePackage: github.com/uber/zanzibar/examples/example-gateway/build/gen-code
 genMock: true
+clientInterfacesMock:
+  github.com/uber/zanzibar/examples/example-gateway/build/clients/contacts:
```

Comment:
add a client with a different interface name to test if mockable and if a fallback is not being used.